File: Find Transition Point/solution1.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution
{
public:    
    
    int transitionPoint(int arr[], int n) {
        // code here
        for(int i=0;i<n;i++) {
            if(arr[i] == 1) return i;
        }
        return -1;
    }
};
int main() 
{
    int n;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++) cin>>arr[i];
    Solution sol;
    cout<<sol.transitionPoint(arr,n);
}

File: Multiply two strings/solution1.java
import java.math.BigInteger;

class Solution
{
    public String multiplyStrings(String s1,String s2)
    {
        //code here.
        BigInteger result = new BigInteger(s1).multiply(new BigInteger(s2));
        return result.toString();
    }
}
public class solution1 {
    public static void main(String[] args) {
        String s1 = "123456";
        String s2 = "1234567";
        Solution sol = new Solution();
        String result = sol.multiplyStrings(s1, s2);
        System.out.println(result);
    }
}
